.github/workflows/rust.yml

```diff
@@ -46,8 +46,10 @@ jobs:
         env:
           RUST_BACKTRACE: 1
         run: |
+          sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 25088A0359807596
```

Comment:
Tbh I'd rather just update to the beta runners
If those become stable it just continues working, whilst this seems more issue prone

Reply:
what about glibc

Reply:
Fair enough
I'd suggest putting the Rust CI as beta runners, since we don't care for the artifacts
And only use the workaround for a build on stable runners for releases

Reply:
@The-personified-devil I mostly agree with you. Since we don't really share commands between runners, it's will be the most succinct solution. But better yet would be to be able to share commands somehow. Maybe making our own gh action? Not having something like subroutines is quite limiting.

Reply:
https://docs.github.com/en/actions/creating-actions/creating-a-composite-action this is what we want, but I think that should be part of my CI update PR and not this

Reply:
tbh i would rather push towards using cross

Reply:
> https://docs.github.com/en/actions/creating-actions/creating-a-composite-action this is what we want, but I think that should be part of my CI update PR and not this

This sounds good. We could use this for quite a few things